tests/unit/helpers/mocks.py

```diff
@@ -54,8 +54,9 @@ def __init__(self, *args, **kwargs):
 
 
 class MockFiles(object):
-    def __init__(self, data):
+    def __init__(self, data, mimetype='application/pdf'):
```

Comment:
Yeah, but `MockFileObject` are instantiated indirectly down the road... how can I pass them this information otherwise?

Why it has to be complex to test :cry: ?